akd/Cargo.toml

```diff
@@ -4,24 +4,20 @@ version = "0.9.0-pre.1"
 authors = ["Harjasleen Malvai <hmalvai@fb.com>", "Kevin Lewi <klewi@fb.com>", "Sean Lawlor <seanlawlor@fb.com>"]
 description = "An implementation of an auditable key directory"
 license = "MIT OR Apache-2.0"
-edition = "2018"
+edition = "2021"
 keywords = ["key-transparency", "akd"]
-repository = "https://github.com/novifinancial/akd"
+repository = "https://github.com/facebook/akd"
 readme = "../README.md"
 
 [features]
-# Supported hash functions
-sha512 = ["akd_core/sha512"]
-sha256 = ["akd_core/sha256"]
-sha512_256 = ["akd_core/sha512_256"]
-sha3_256 = ["akd_core/sha3_256"]
-sha3_512 = ["akd_core/sha3_512"]
-blake3 = ["akd_core/blake3"]
+# Supported configurations
+whatsapp_v1 = ["akd_core/whatsapp_v1"]
+experimental = ["akd_core/experimental"]
 
-bench = ["blake3", "public_tests","tokio/rt-multi-thread"]
-public_tests = ["rand", "bincode", "colored", "once_cell", "serde_serialization", "akd_core/rand"]
-public_auditing = ["protobuf", "akd_core/protobuf"]
-serde_serialization = ["serde", "ed25519-dalek/serde", "akd_core/serde_serialization"]
+bench = ["experimental", "public_tests", "tokio/rt-multi-thread", "dep:paste"]
+public_tests = ["dep:rand", "dep:bincode", "dep:colored", "dep:once_cell", "serde_serialization", "akd_core/rand", "dep:paste"]
+public_auditing = ["dep:protobuf", "akd_core/protobuf"]
```

Comment:
Do you have documentation on this `dep:` syntax? I've never seen this

Reply:
See https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies -- without the dep, any dependency marked as "optional" is automatically a toggle-able feature. By using dep, we are saying that it is an optional dependency that should not be a feature.